autoagents/train/scripts/action_finetuning_v3.sh

```diff
@@ -0,0 +1,26 @@
+python3 -m torch.distributed.run --nproc_per_node=8 \
+    --master_port=20005 --nnodes=1 --node_rank=$RANK --master_addr=127.0.0.1 \
+    autoagents/train/train_v3.py \
+    --model_name_or_path input-model \
+    --data_path json-data.json \
+    --bf16 True \
+    --output_dir output-path \
+    --num_train_epochs 2 \
+    --per_device_train_batch_size 1 \
```

Comment:
Note that we can probably bump this up to 4 or 16, decreasing `--gradient_accumulation_steps` appropriately